src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  font-family: "Inter", sans-serif;
```

Comment:
Consider using a variable for the font-family to make it more reusable and maintainable. For example, `$fontFamilyInter: 'Inter', sans-serif;`. 🎨